nerves_defconfig

```diff
@@ -49,10 +50,18 @@ BR2_PACKAGE_FWUP=y
 BR2_PACKAGE_LINUX_FIRMWARE=y
 BR2_PACKAGE_RPI_FIRMWARE=y
 BR2_PACKAGE_RPI_FIRMWARE_X=y
+BR2_PACKAGE_DBUS_GLIB=y
 BR2_PACKAGE_RPI_USERLAND=y
+BR2_PACKAGE_SETSERIAL=y
 BR2_PACKAGE_ERLANG_SMP=y
 BR2_PACKAGE_LIBMNL=y
-BR2_PACKAGE_NCURSES=y
+BR2_PACKAGE_BLUEZ5_UTILS=y
+BR2_PACKAGE_BLUEZ5_UTILS_OBEX=y
+BR2_PACKAGE_BLUEZ5_UTILS_CLIENT=y
+BR2_PACKAGE_BLUEZ5_UTILS_GATTTOOL=y
+BR2_PACKAGE_BLUEZ5_UTILS_EXPERIMENTAL=y
+BR2_PACKAGE_DNSMASQ=y
```

Comment:
Ah, yes, that makes sense: The dependency is the other way round. I will remove `setserial`, it is not needed for this case.